components/EventCard/styles.ts

```diff
@@ -1,28 +1,38 @@
 
 import { StyleSheet } from 'react-native'
-import { lightGray, peach, shadowColor} from '../../assets/colors/colors'
+import { lightGray, lightPeach, peach, shadowColor} from '../../assets/colors/colors'
 
 export const styles = StyleSheet.create({
   cardContainer: {
-    backgroundColor: peach,
+    backgroundColor: lightPeach,
+    borderColor: peach,
     borderRadius: 10,
-    elevation: 5,
-    margin: 10,
-    overflow: 'hidden',
+    borderWidth: 2,
+    flex: 1,
+    height: 202,
+    margin: 5,
+    overflow: 'scroll',
+    
     shadowColor: shadowColor,
     shadowOffset: {
       height: 2,
       width: 0,
     },
     shadowOpacity: 0.25,
     shadowRadius: 3.84,
+    width: 172,
+    
   },
   description: {
+    
     color: lightGray,
     fontSize: 14,
+    fontStyle: 'italic',
+    textAlign: 'right',
```

Comment:
You seem to also forget to use our own custom font. Don't forget to define it with a `fontFamily` parameter